internal/pkg/service/appproxy/http/router.go

```diff
@@ -95,65 +96,87 @@ func (r *Router) CreateHandler() http.Handler {
 	return mux
 }
 
-func (r *Router) createDataAppHandler(app DataApp) (http.Handler, error) {
+func (r *Router) createForbiddenHandler() http.Handler {
+	return http.HandlerFunc(func(w http.ResponseWriter, req *http.Request) {
+		w.WriteHeader(http.StatusForbidden)
+		fmt.Fprintln(w, "Application has misconfigured OAuth2 provider.")
+	})
+}
```

Comment:
`forbiddenHandler` -> `configErrorHandler`?  forbidden sounds to me like it's the user's fault